src/test/java/calculator/CalculratorValidation.java

```diff
@@ -0,0 +1,62 @@
+package calculator;
+
+import calculator.separator.CustomSeparator;
+import org.thymeleaf.expression.Numbers;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static calculator.separator.CustomSeparator.*;
+
+/**
+ * <pre>
+ * calculator
+ *      CalculratorValidation
+ * </pre>
+ *
+ * @author YunJin Choi(zzdd1558@gmail.com)
+ * @since 2022-03-02 오전 1:34
+ */
+
+public class CalculratorValidation {
+
+    public static boolean isNullOrEmpty(final String text) {
+        return isNull(text) || text.trim().isEmpty();
+    }
+
+    public static boolean isNull(final String text) {
+        return Objects.isNull(text);
+    }
+
+    public static boolean isSingleNumber(String text) {
+        try {
+            Integer.parseInt(text);
+            return true;
+        }catch (NumberFormatException e) {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        } catch (NumberFormatException e) {
```